specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-02-07-preview/examples/ProviderInstances_Create.json

```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+      "api-version": "2020-02-07-preview",
+      "subscriptionId": "00000000-0000-0000-0000-000000000000",
+      "resourceGroupName": "myResourceGroup",
+      "sapMonitorName": "mySapMonitor",
+      "providerInstanceName": "myProviderInstance",
```

Comment:
Do we need this providerInstancename twice?

Reply:
twice?
`providerInstanceName` is the name of the parameter
`"myProviderInstance"` is the name of the sample provider instance

Reply:
I mean we have the name under the providerInstanceParameter as well. Do we need the name field in the providerInstanceParameter?

Reply:
oic what you mean.
I think that is standard practice, both our current HANA instance and [AKS](resource-manager/Microsoft.HanaOnAzure/preview/2020-02-07-preview/examples/ProviderInstances_Create.json) is the same